Strategies/ITVision.md

```diff
@@ -0,0 +1,134 @@
+# ESDC's IT Vision
+
+[HDLC](HumanDevelopmentLifeCycle.md) -&gt; [KPIs](IITB_KPIs.md) -&gt; IT Vision -&gt; Guiding Principles -&gt; IT Strategies (roadmap)
+
+The following is ESDC's IT Vision.
+It paints a vision of the future of IT within ESDC.
+It is **not** meant to be time bound, but rather as an ideal state to which ESDC IT needs to strive to get to.
+Such state will continuously evolve when new information and knowledge is captured, re-adjusting the vision in the process.
+
+The IT Vision complements the [HDLC](HumanDevelopmentLifeCycle.md) story which is taken from the point of view of the citizen, and the [KPIs](IITB_KPIs.md) which define the performance metrics needed to sustain the HDLC, by providing the point of view of the organization's IT capabilities.
+
+The IT Vision leverages the work done in the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), ensures alignment with [IITB's high level priorities](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml), and brings the [GC's Strategic Plan for IM/IT 2017-2021](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) into ESDC's operational context.
+
+The ESDC IT Vision is explained using the [GC's Strategic Plan for IM/IT 2017-2021](https://www.canada.ca/en/treasury-board-secretariat/services/information-technology/strategic-plan-2017-2021.html#toc4) six vision pillars:
+
+1. Digital Services
+2. Open and accessible
+3. Security
+4. Enterprise management
+5. Community
+6. Workplace
+
+## 1. Digital Services
+
+All of [ESDC's services](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) are delivered through technology with a large majority running on the public cloud to seek benefits of rapid elasticity, broad internet access, and up to date security patching.
```

Comment:
what do you mean by broad internet?

Reply:
@e-wu regards bandwidth and accessibility by mobile devices. Trying to simplify greatly the [NIST definition](https://csrc.nist.gov/publications/detail/sp/800-145/final)

> Broad network access. Capabilities are available over the network and accessed through standard mechanisms that promote use by heterogeneous thin or thick client platforms (e.g., mobile phones, tablets, laptops, and workstations).

Reply:
@e-wu should this be reworded to clarify? 
Though I worry to have a definition that's to ambiguous and long (like NIST's)

Reply:
linked to NIST definition regarding meaning behind 
> rapid elasticity, broad internet access, and up to date security patching